cdap-ui/app/features/etlapps/controllers/create-ctrl.js

```diff
@@ -196,6 +196,19 @@ angular.module(PKG.name + '.feature.etlapps')
       });
     };
 
+    $scope.deleteTrasformProperty = function(transform) {
+      var filterFilter = $filter('filter');
+      var index = $scope.transforms.indexOf(transform);
+      $scope.transforms.splice(index, 1);
```

Comment:
No. We show the list and we delete it. So if the user clicked on the item to be deleted from an array then we are sure that its there.